support/build/php

```diff
@@ -109,6 +109,7 @@ if [[ $dep_version == 7.* ]]; then
     elif [[ $STACK != "cedar-14" ]]; then
         configureopts="\
```

Comment:
@dzuelke isn't this code loosing the `--enable-opcache-file` which was first added  for PHP 7+ above ?

Reply:
This was already there 🤔

Reply:
yeah, I know this is not related to your own change. Your own change seems good regarding existing code (if there is a bug, it was already there before)